src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStepTest.java

```diff
@@ -288,6 +288,40 @@ public DescriptorImpl() {
         
     }
 
+    @Issue("JENKINS-28822")
+    @Test public void interruptingAbortsBuild() throws Exception {
+        WorkflowJob p = j.jenkins.createProject(WorkflowJob.class, "p");
+        p.setDefinition(new CpsFlowDefinition("node {\n" +
+                "  timeout(time: 1, unit: 'SECONDS') {" +
+                (Functions.isWindows()
+                        ? "bat 'ping -n 6 127.0.0.1 >nul'\n"
+                        : "sh 'sleep 5'\n") +
+                "  }" +
+                "}", true));
+        WorkflowRun b = p.scheduleBuild2(0).waitForStart();
+        j.waitForCompletion(b);
+        // Fails with Result.FAILURE as of commit a224295d12.
```

Comment:
?
Is this a C&P'ed comment?

Reply:
No, but it looks like I used a commit hash from durable-task (https://github.com/jenkinsci/durable-task-plugin/commit/a224295d12), which is wrong. I'll add a PR to fix it, thanks for noticing!

Reply:
See #83.